misc/git-hook/optional/precommit/modify/spaces2tabs.bash

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
Hmm not sure how i didn't notice this before.

I know i said i prefer tabs before. But then i saw that different browsers show the git diff differently when tabs or spaces are used.
Which leads me to think it'd be nice to conserve spaces instead of tabs.
This is due to the fact that tabs are rendered on different widths in different programs whilst spaces have a fixed width.

Within the editor of your choice you can set to show spaces as tabs if it meets a certain value.

After installing these scripts we'll run into problems when converting from tabs to spaces for the files you've changed. But i think that's alright. Not sure if @OznOg or @Lord-Kamina have any concerns regarding this?

Reply:
Majority of the code base is tabs if that makes for an easier decision on this